src/components/Organisms/AlertManager/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from "react";
+
+import { useAlertHandlerContext } from "../../contexts/alert_handler";
+
+import Alert from "../../Atoms/Alert/index.jsx";
+
+const AlertManager = () => {
+  const { visible, message, closeAlert } = useAlertHandlerContext();
+
+  useEffect(() => {
+    if (visible) {
+      const timer = setTimeout(() => {
+        closeAlert();
+      }, 5000);
```

Comment:
```suggestion
const ALERT_DURATION_MS = 5000;

const AlertManager = () => {
  const { visible, message, closeAlert } = useAlertHandlerContext();

  useEffect(() => {
    if (visible) {
      const timer = setTimeout(() => {
        closeAlert();
      }, ALERT_DURATION_MS);
```

こんな感じで定数化すると可読性が上がると言われています！
マジックナンバーをへらすということを意識してみるのがおすすめです。